website/docs/cli/commands/terraform/usage.mdx

```diff
@@ -114,6 +114,8 @@ atmos terraform workspace test/test-component-override-3 -s tenant1-ue2-dev --re
 atmos terraform workspace test/test-component-override-3 -s tenant1-ue2-dev --redirect-stderr ./errors.txt
 
 atmos terraform plan test/test-component -s tenant1-ue2-dev -- -refresh=false -lock=false
+
+atmos terraform plan test/test-component -s tenant1-ue2-dev --append-user-agent "Atmos/1.0.0 (Cloud Posse; +https://atmos.tools)"
```

Comment:
The use-case is that companies want to set this to their own metadata. Less so to change how Atmos reports it's version.

```suggestion
atmos terraform plan test/test-component -s tenant1-ue2-dev --append-user-agent "Acme/1.0 (Build 1234; arm64)"
```

Reply:
agree, this is done